src/main/java/com/authlete/common/api/AuthleteApi.java

```diff
@@ -441,7 +567,89 @@ public interface AuthleteApi
      *
      * @since 2.29
      */
-    TokenListResponse getTokenList(String clientIdentifier, String subject, int start, int end) throws AuthleteApiException;
+    TokenListResponse getTokenList(String clientIdentifier, String subject,
+                                   int start, int end) throws AuthleteApiException;
+
+
+    /**
+     * Get the list of access tokens
+     * (= call Authlete's {@code /auth/token/get/list} API with {@code clientIdentifier},
+     * {@code subject}, {@code start} and {@code end} parameters).
+     *
+     * <p>
+     * When both {@code clientIdentifier} and {@code subject} are
+     * {@code null}, the list of access tokens that are associated
+     * with the service is returned.
+     * </p>
+     *
+     * <p>
+     * When {@code clientIdentifier} is {@code null} but the {@code subject}
+     * is not {@code null}, the list of access tokens that are associated
+     * with the subject is returned.
+     * </p>
+     *
+     * <p>
+     * When {@code clientIdentifier} is not {@code null} but the
+     * {@code subject} is {@code null}, the list of access tokens
+     * that are associated with the client application is returned.
+     * </p>
+     *
+     * <p>
+     * When neither {@code clientIdentifier} nor {@code subject} is
+     * {@code null}, the list of access tokens that are associated
+     * with the client application and the subject is returned.
+     * </p>
+     *
+     * <p>
+     * The pair of {@code start} and {@code end} parameters denotes
+     * the range of the result set of the query. For example, if
+     * {@code start} is 5 and {@code end} is 7, the pair makes a
+     * range from 5 (inclusive) to 7 (exclusive) and the response
+     * will contain (at most) 2 pieces of access token information,
+     * i.e., information about the 6th and the 7th access tokens (the
+     * index starts from 0).
+     * </p>
+     *
+     * <p>
+     * If {@code end - start} is equal to or less than 0, {@link
+     * TokenListResponse#getAccessTokens() getAccessTokens()} method
+     * of the response returns {@code null}. But even in such a case,
+     * {@link TokenListResponse#getTotalCount() getTotalCount()}
+     * method returns the total count. In other words, if you want
+     * to get just the total count, you can write the code as
+     * shown below.
+     * </p>
+     *
+     * <pre>
+     * int totalCount = api.{@link #getTokenList(String, String, int, int, TokenStatus)
+     * getTokenList(clientIdentifier, subject, 0, 0)}.{@link TokenListResponse#getTotalCount()
```

Comment:
`"getTokenList(String, String, int, int, TokenStatus)` does not match `getTokenList(clientIdentifier, subject, 0, 0)`.